PROTO_tests/tests/parquet_test.py

```diff
@@ -0,0 +1,695 @@
+import os
+import tempfile
+
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+import pytest
+from pandas.testing import assert_series_equal
+
+import arkouda as ak
+from arkouda import io_util
+
+TYPES = ("int64", "uint64", "bool", "float64", "str")
+COMPRESSIONS = ["snappy", "gzip", "brotli", "zstd", "lz4"]
+SIZE = 100
+NUMFILES = 5
+verbose = True
+
+
+@pytest.fixture
+def par_test_base_tmp(request):
+    par_test_base_tmp = "{}/.df_test".format(os.getcwd())
+    io_util.get_directory(par_test_base_tmp)
+
+    # Define a finalizer function for teardown
+    def finalizer():
+        # Clean up any resources if needed
+        io_util.delete_directory(par_test_base_tmp)
+
+    # Register the finalizer to ensure cleanup
+    request.addfinalizer(finalizer)
+    return par_test_base_tmp
+
+
+class TestParquet:
```

Comment:
similar to import export tests, I think these should be covered by a class in `io_test`
https://github.com/Bears-R-Us/arkouda/blob/45a07b36df356f4614b3a94bf942c8abe3a8fc50/PROTO_tests/tests/io_test.py#L143

Reply:
OK, I combined `parquet_test` and `io_test.py`.  It seemed like there were a few tests that weren't covered and needed to be moved over.